Mage.Client/src/main/java/mage/client/deckeditor/DeckEditorPanel.java

```diff
@@ -759,6 +759,7 @@ private void importFromFile(java.awt.event.ActionEvent evt) {
             MageFrame.getDesktop().setCursor(new Cursor(Cursor.WAIT_CURSOR));
             try {
                 DeckImporter importer = DeckImporter.getDeckImporter(file.getPath());
+                importer.setPreferences(MageFrame.getPreferences());
```

Comment:
Must change the logic: import original cards, press right click to add/remove/apply favorite sets/cards versions. Do not modify import code with hidden features.